frontend/src/components/RosalutionHeader.vue

```diff
@@ -63,6 +73,11 @@ export default {
         return [];
       },
     },
+    third_party_links: {
```

Comment:
As mentioned in the comment, https://github.com/uab-cgds-worthey/rosalution/pull/70/files/963ffffcea90c0a4adbc431b820feda868a6ba7d#r1172812462 the third_party_links array, should match the style of the actions menu prop as seen above in lines 71 - 75.    

Also, this will be good to have as a reference in the future, but props are optional by default and do not need to be indicated false.  
https://vuejs.org/guide/components/props.html#prop-validation

> All props are optional by default, unless required: true is specified.

Reply:
Thanks. I think I have made these changes.